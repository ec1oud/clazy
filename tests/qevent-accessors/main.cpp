#include <QtCore/QObject>
#include <QtGui/QMouseEvent>
#include <QtWidgets/QWidget>

class TestMouseEvent : public QWidget
{
public:
    TestMouseEvent();
    void doSomethingWith(const QPoint &) { }
    void mouseMoveEvent(QMouseEvent *event) override {
        doSomethingWith(event->pos()); // Warn
    }
};

