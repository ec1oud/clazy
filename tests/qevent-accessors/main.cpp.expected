qevent-accessors/main.cpp:11:32: warning: Replace the deprecated call QMouseEvent::pos(): use position().toPoint() instead [-Wclazy-qevent-accessors]
